Makefile

```diff
@@ -11,8 +11,13 @@ export TERRAFORM_PROVIDER_VERSION := 1.24.0
 export TERRAFORM_PROVIDER_DOWNLOAD_NAME := terraform-provider-rancher2
 export TERRAFORM_PROVIDER_DOWNLOAD_URL_PREFIX := https://github.com/rancher/terraform-provider-rancher2/releases/download/v$(TERRAFORM_PROVIDER_VERSION)
 
+# Not supported by this terraform provider atm
+export TERRAFORM_NATIVE_PROVIDER_BINARY := 
```

Comment:
why is it not supported? Can't we guess/predict the provider name?

Reply:
The original intent of the native binary change to the Dockerfile by the author is not clear to me. Unless it was a fix for terraform providers bundling multiple binaries in the same zip. But I have not found any evidence of this in the `null` and `azurerm` providers.

As far as I understand it, it boils down to the terraform plugin sdk version used to build a provider and the v4 -> v5 (grpc) protocol changes made from sdk v1 to sdk v2.

terraform-provider-azurerm:
- go.mod: `github.com/hashicorp/terraform-plugin-sdk/v2 v2.18.0`
- published zip with provider binary extracts a `..._v5` binary, a tf plugin/provider using protocol v5 (grpc)

terraform-provider-null (from template):
- go.mod: `github.com/hashicorp/terraform-plugin-sdk/v2 v2.18.0`
- published zip extracts a `...v5` binary

terraform-provider-rancher2:
- go.mod: `github.com/hashicorp/terraform-plugin-sdk v1.14.0`
- published zip extract a binary without any protocol indication (not _v4 nor _v5)

Hence my remark 'native' (grpc or better said protocol v5) is not supported by terraform-provider-rancher2 as it is build against SDK v1 not SDK v2.

But like I said, I am not sure why all this 'native binary' plumbing was introduced in the first place. If you d/l the zip file, it extracts a single binary, the protocol version (v4 or v5 or v6) is not really relevant, just place that single extracted binary in the right tf plugin dir. Only if tf core drops support for protocol v4 we are stuck with an older tf core version, unless terraform-provider-rancher2 upgrades to SDK v2.

Reply:
ok cool, so maybe just remove this variable as it is not used at all?